tests/tests_unit/test_cognite_client.py

```diff
@@ -54,6 +56,22 @@ def mock_token_inspect(rsps) -> None:
     yield
 
 
+@pytest.fixture(scope="function")
+def set_env_vars(monkeypatch):
+    with patch.dict(os.environ, clear=True):
+        env_vars = {
+            "COGNITE_PROJECT": "test-project",
+            "COGNITE_CLIENT_NAME": "test-project",
+            "credential_type": "o_auth_client_credentials",
+            "URL": "test",
+            "COGNITE_CLIENT_SECRET": "test-client-secret",
+            "COGNITE_DEBUG": "true",
+        }
+        for key, value in env_vars.items():
+            monkeypatch.setenv(key, value)
+        yield
```

Comment:
I think you can simplify this a bit:
```suggestion
@pytest.fixture
def set_env_vars(monkeypatch):
    env_vars = {
        "COGNITE_PROJECT": "test-project",
        "COGNITE_CLIENT_NAME": "test-project",
        "credential_type": "o_auth_client_credentials",
        "URL": "test",
        "COGNITE_CLIENT_SECRET": "test-client-secret",
        "COGNITE_DEBUG": "true",
    }
    monkeypatch.setattr(os, 'environ', env_vars)
```